ase/sw/ipc_mgmt_ops.c

```diff
@@ -38,6 +38,9 @@
 // IPC control list
 static char ipclist_filepath[ASE_FILEPATH_LEN];
 
+// Local IPC log
+FILE *local_ipc_fp;
```

Comment:
Unrelated change

Reply:
Agree. This should be in a separate commit.

Reply:
I can make two commits. One commit for wsmeta_t changes and another commit for the rest.